nuvla/job_engine/job/actions/utils/bulk_deployment_set_apply.py

```diff
@@ -87,9 +87,17 @@ def _update_api_endpoint(self, deployment):
             deployment['api-endpoint'] = self.api_endpoint
 
     def _get_infra(self, target):
+        deployment_set = self.dg_api.get(self.dep_set_id)
```

Comment:
Is it possible to not retrieve here deployment set because we retrieve it in another place? Maybe store it in self and reuse here